nibabel/ecat.py

```diff
@@ -155,10 +155,10 @@
 _dtdefs = ( # code, name, equivalent dtype
     (1, 'ECAT7_BYTE', np.uint8),
     (2, 'ECAT7_VAXI2', np.int16),
-    (3, 'ECAT7_VAXI4', np.float32),
+    (3, 'ECAT7_VAXI4', np.int32),
     (4, 'ECAT7_VAXR4', np.float32),
     (5, 'ECAT7_IEEER4', np.float32),
-    (6, 'ECAT7_SUNI2', np.uint16),
+    (6, 'ECAT7_SUNI2', np.int16),
```

Comment:
The pointer starts out as a `char *` and isn't re-cast as anything else, which makes me think it is unsigned. What made you think it was signed?